tests/Testcontainers.Tests/Unit/Configurations/DockerEndpointAuthenticationProviderTest.cs

```diff
@@ -35,7 +37,17 @@ public AuthProviderTestData()
       {
         var isWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
         var defaultConfiguration = new PropertiesFileConfiguration(Array.Empty<string>());
-        var dockerHostConfiguration = new PropertiesFileConfiguration(new[] { "docker.host=" + DockerHost });
+        var dockerHostConfiguration = new PropertiesFileConfiguration(new[] { $"docker.host={DockerHost}" });
+        var dockerTlsConfiguration = new PropertiesFileConfiguration(new[] { "docker.tls=true" });
+        var dockerMTlsConfiguration = new PropertiesFileConfiguration(new[] { "docker.tls.verify=true" });
+        this.Add(new object[] { new MTlsEndpointAuthenticationProvider(defaultConfiguration), false });
+        // this.Add(new object[] { new MTlsEndpointAuthenticationProvider(dockerMTlsConfiguration), true });
```

Comment:
I need to fix those 2 tests. They require the cert and key file. I think we can generate a temp dummy file for that.

Reply:
Yep sounds good. I had that specific code like youhave right now but I removed it.